dashboard/test/controllers/levels_controller_test.rb

```diff
@@ -3,6 +3,8 @@
 class LevelsControllerTest < ActionController::TestCase
   include Devise::Test::ControllerHelpers
 
+  default_update_blocks_params = nil
```

Comment:
This won't be accessible anywhere. Member variables in ruby must be preceded with a `@`, code here in the class won't be executed when the class is loaded, not instantiated.

Since `setup` runs before each test case you don't need to initialize the member, just define it directly. If you want to use it without the `@` you can add an accessor with `attr_reader :default_update_blocks_params`, or instead of in setup define it as a private method so it's lazily constructed as in this [example](https://github.com/code-dot-org/code-dot-org/blob/19d6403d6035f4a3cfa101a5a10bd23e8627304b/dashboard/app/controllers/pd/workshop_enrollment_controller.rb#L242-L241).